client/src/AddFriend/AddFriend.js

```diff
@@ -7,7 +7,7 @@ import CountDisplay from './CountDisplay'
 import Settings from './Settings'
 import RemoveFriendButton from './RemoveFriendButton'
 import FormAlert from './FormAlert'
-import { alertTable } from './constants'
+import { alertTable } from '../Misc/constants'
```

Comment:
We can just have `..constants/constants` (ie call the folder `constants`) or `../constants` (ie don't make a folder for it)